### DISCUSSION TITLE
Serving static files on vercel using express js not working

### DISCUSSION THREAD

**Discussion by vk-develops** [vercel/community#6497] [Help]
### Summary

I have created a MERN app with two folder Frontend and Backend the frontend folder consist of the complete frontend part using the expo react native. And the Backend part is the node js part.

This is my folder structure
|App
   |--Backend
       |--Public
           |--Images
               |--WeatherImages
                   |--WeatherMist-Day.png
                   |---few more images
               |--Login-Bg.png
       |--index.js
       |--package.json
       |--vercel.json
   |--Frontend
       |--All frontend files
       
And when i use this vercel.json
{
    "version": 2,
    "builds": [
        {
            "src": "/index.js",
            "use": "@vercel/node"
        }
    ],
    "routes": [
        {
            "src": "/api/v1(.*)",
            "dest": "/index.js"
        },
        {
            "src": "/Public/(.*)",
            "dest": "/Public/$1"
        },
        {
            "src": "/(.*)",
            "dest": "/index.js"
        }
    ]
}

the backend api parts are working perfectly but the images are being properly served it throws an error as cannot GET /image urls

and when i use this vercel.json 
{
        "routes": [
        {
            "src": "/Public/(.*)",
            "dest": "/Public/$1"
        },
    ]
}

the images are being correctly served but the api's are not woking and on the "/" path it displays the index.js codes instead of the response message

and i have additionally used these on my index.js file
// Serving static files from the 'Public' directory
const __dirname = path.resolve();
app.use(express.static(path.join(__dirname, "Public")));

app.use("/Images", express.static(path.join(__dirname, "Public", "Images")));
app.use(
    "/Images/WeatherImg",
    express.static(path.join(__dirname, "Public", "WeatherImages"))
);

I need to make the api's work fine and also need to serve the images 

please help me resolve this issue and thanks in advance!
     

### Example

https://github.com/vk-develops/Skysavvy-MERNN-Weather-Forecast-App

### Steps to Reproduce

https://github.com/vk-develops/Skysavvy-MERNN-Weather-Forecast-App

**Comment by Offensic** [comment]
I have the same issue. Did your solve it? I upload images during production on the server using Multer in a Node.js server. It works successfully and shows images in production, but when I deploy my app on Vercel, the images do not show. The API is working perfectly; it's just the images that are not showing on the frontend React app other parts of servr is working perfectly

[a comment by an automated account is not shown]
